### ISSUE TITLE
Page scrolling when zooming with mousewheel in Firefox

### ISSUE THREAD

**Issue by mheppner** [swimlane/ngx-graph#231]
**Describe the bug**
When using ngx-graph as a component within a page, scrolling the mousewheel to zoom in/out also causes the entire page to scroll.

**Expected behavior**
The page should not scroll when using the mouse wheel within the bounds of the ngx-graph component.

I see that the component is calling `preventDefault()` in the [mouse wheel](https://github.com/swimlane/ngx-graph/blob/5.5.0/src/graph/mouse-wheel.directive.ts) directive for the `mousewheel`, `DOMMouseScroll`, and `onmousewheel` events, but this still doesn't seem to work for Firefox 68.

**ngx-graph version**
5.5.0

**Additional context**
Adding my own event listeners to my component, Firefox is picking up both the `DOMMouseScroll` event and the `MozMousePixelScroll` event. Calling `preventDefault()` on `MozMousePixelScroll` seems to fix the issue.

**Comment by marjan-georgiev** [comment]
Where are you able to reproduce this? I am trying on the demo page with firefox 68 and it doesn't see to happen.

**Comment by mheppner** [comment]
It's also happening for me on the demo pages. Firefox 68.0.1 on MacOS Mojave with a 2015 MacBook Pro, using either the trackpad or the magic mouse. I tried disabling inertial scrolling, but the same event is still firing. Also confirmed on my coworker's laptop, Firefox 68, MacOS Mojave, 2018 MacBook Pro.

Some evidence about the scroll events on Macs:
- https://stackoverflow.com/a/27379942
- https://github.com/jquery/jquery-mousewheel/issues/45#issuecomment-11749359
- https://stackoverflow.com/a/46612031

I can verify this with this pen: https://codepen.io/anon/pen/YmQmdj?editors=1111
The events firing for me on Firefox are `wheel`, `DOMMouseScroll`, `MozMousePixelScroll`, all at various frequencies.

**Comment by mheppner** [comment]
Hm, interesting outcome, but `preventDefault()`ing just the `wheel` event is enough to stop the others.

I think there's confusion on the event names. `onmousewheel` is the correct deprecated event, and `mousewheel` is just what it's called? Look in the blue box for the [event handler property](https://developer.mozilla.org/en-US/docs/Web/API/Element/mousewheel_event). Searching for just `mousewheel` shows that all of the handler references are to `onmousewheel`.

Basically, I think `mousewheel` probably isn't capturing anything from any browser, but `wheel` event is the correct modern one to use.

**Comment by assafsun** [comment]
Something is not clear to me.
On Firefox version 68, the scroll and zoom works fine.
Firefox is not support wheel event but do support scroll so it should work.

Did you tried to see if it reproduce on the latest demo application and as a result on version 6 (instead on 5.5)? Your PR is against master branch which is version 6.
I tried also to reproduce it and didn't succeeded, In addition, I'm working on some track pad improvements for ngx-graph and I didn't noticed this problem.

**Comment by mheppner** [comment]
The scroll wheel directive hasn't changed much between v5 and v6. The currently deployed demo pages also have the same issue, which I'm assuming are using the latest v6.

Did you try to reproduce this on a Mac? I believe FF on Mac is throwing more events. I've confirmed this behavior on several MacBooks at work.

I believe the [`wheel`](https://developer.mozilla.org/en-US/docs/Web/API/Element/wheel_event) should be captured, which is in fact supported by Firefox. The MDN pages indicated that `wheel` is the modern event that should be used (not being captured at all in ngx-graph's directive). Deprecated events [`mousewheel`](https://developer.mozilla.org/en-US/docs/Web/API/Element/mousewheel_event) (already captured, Chrome) and [`DOMMouseScroll`](https://developer.mozilla.org/en-US/docs/Web/API/Element/DOMMouseScroll_event) (already captured, FF) should still be in the directive, but `onmousewheel` should not be, since that's the property name on the window object and not the event; `onmousewheel` will never be captured using `addEventListener()`.

I updated the PR to reflect these changes.
